### ISSUE TITLE
Local storage

### ISSUE THREAD

**Issue by colemarkham** [seemikehack/scorewarden#11]
Save to local storage to avoid clearing on refresh

**Comment by seemikehack** [comment]
Of course, just discovered a bug, but I don't want to deal with it right now, so here goes:

Once you've got some players on the scoreboard, they never go away...because I'm not listening to remove events on the PlayerViews and they are thus never removed from memory and thus never removed from local storage. So there's that.

On an obliquely related note, the solution method for this issue could likely have been used on the add player migration from abd2a97 in this thread.
